users/frobiac/keymap_german.h

```diff
@@ -0,0 +1,151 @@
+/* Copyright 2015-2016 Matthias Schmidtt
```

Comment:
This file should be removed and `./quantum/keymap_extras/keymap_german.h` used instead.

Reply:
Right - as a side-effect of using default keymaps instead of userspace-named ones, this now also works without this cludge. Thanks for being so thorough, would have missed this obvious requirement.